src/common/snippets/tests/src/lowering_utils.cpp

```diff
@@ -34,6 +34,13 @@ DummyTargetMachine::DummyTargetMachine() {
     jitters[ngraph::snippets::op::TileScheduler::get_type_info_static()] = dummy_functor;
 }
 
+LoweringTests::LoweringTests() : TransformationTestsF() {
+    // external subgraph input shape and internal parameters shapes
+    // might differ due to the blocked layout
+    // so input & output descriptors shouldn't be checked
+    comparator.disable(FunctionsComparator::CmpValues::SUBGRAPH_DESCRIPTORS);
```

Comment:
This change connected to the `snippets::Subgraph` inheritance  from `MultiSubGraph` class. Because of that the graph_comparator checks input/output descriptors. The Graph comparator expects that an external Subgraph input and the corresponding internal Parameter have the same shapes, but due to the `Subgraph` internal logic this condition isn't met.
So the decision is to avoid such checks in Lowering tests